tests/test_content_loader.py

```diff
@@ -836,6 +844,19 @@ def test_get_data(self):
         ])
         assert section.get_data(form)['q4'] == ['']
 
+        # if we have one empty value
+        form = ImmutableMultiDict([
+            ('q5-0', '')
+        ])
+        assert section.get_data(form)['q5'] == ['']
```

Comment:
Should this be a `None` rather than an empty string? Or are we leaving non-truthy strings as whatever string got submitted?